api/users.go

```diff
@@ -174,7 +175,7 @@ func (a *API) verifyUserAccountHandler(w http.ResponseWriter, r *http.Request) {
 // returned.
 func (a *API) userVerificationCodeInfoHandler(w http.ResponseWriter, r *http.Request) {
 	// get the user email of the user from the request query
-	userEmail := r.URL.Query().Get("email")
+	userEmail := chi.URLParam(r, "email")
```

Comment:
`"email"` could be a constant

Reply:
Hm, not sure I get you. This is not a technique used for the rest of the URI parameters. WHy should we apply it here?